src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -170,26 +171,28 @@ const IntegrationItemBox = styled('div')`
   flex: 1;
 `;
 
-const IntegrationStatus = styled(
-  (props: React.HTMLAttributes<HTMLElement> & {status: ObjectStatus}) => {
-    const {status, ...p} = props;
-    const color = status === 'active' ? theme.success : theme.gray300;
-    const titleText =
-      status === 'active'
-        ? t('This Integration can be disabled by clicking the Uninstall button')
-        : t('This Integration has been disconnected from the external provider');
-    return (
-      <Tooltip title={titleText}>
-        <div {...p}>
-          <CircleIndicator size={6} color={color} />
-          <IntegrationStatusText>{`${
-            status === 'active' ? t('enabled') : t('disabled')
-          }`}</IntegrationStatusText>
-        </div>
-      </Tooltip>
-    );
-  }
-)`
+const IntegrationStatus = (
+  props: React.HTMLAttributes<HTMLElement> & {theme: Theme; status: ObjectStatus}
+) => {
+  const {theme, status, ...p} = props;
+  const color = status === 'active' ? theme.success : theme.gray300;
+  const titleText =
+    status === 'active'
+      ? t('This Integration can be disabled by clicking the Uninstall button')
+      : t('This Integration has been disconnected from the external provider');
+  return (
+    <Tooltip title={titleText}>
+      <div {...p}>
+        <CircleIndicator size={6} color={color} />
+        <IntegrationStatusText>{`${
+          status === 'active' ? t('enabled') : t('disabled')
+        }`}</IntegrationStatusText>
+      </div>
+    </Tooltip>
+  );
+};
+
+const StyledIntegrationStatus = styled(withTheme(IntegrationStatus))`
```

Comment:
wouldn't it better to move `withTheme` to the component?

Reply:
I think you are right, I will test this. Thank you for the feedback.